src/client-entry.ts

```diff
@@ -5,14 +5,16 @@ import TermboxRequest from '@/common/TermboxRequest';
 import { factory } from '@/common/TermboxFactory';
 import UlsLanguageTranslationRepository from '@/client/data-access/UlsLanguageTranslationRepository';
 import EntityRepository from '@/client/data-access/EntityRepository';
-import MwWindow from './client/MwWindow';
+import MwWindow from './client/mediawiki/MwWindow';
```

Comment:
As Pablo mentioned earlier in one of my commits - can we use relative or absolute paths and not mix it? Or at least set up a rule, when we are allowed to use relative paths?

Reply:
Good point, fixed!